imports/client/components/HuntListPage.tsx

```diff
@@ -65,9 +66,11 @@ const Hunt = React.memo(({ hunt }: { hunt: HuntType }) => {
       </ModalForm>
       <ButtonGroup size="sm">
         {canUpdate ? (
-          <Button as={Link} to={`/hunts/${huntId}/edit`} variant="outline-secondary" title="Edit hunt...">
-            <FontAwesomeIcon fixedWidth icon={faEdit} />
-          </Button>
+          <LinkContainer to={`/hunts/${huntId}/edit`}>
+            <Button as="a" variant="outline-secondary" title="Edit hunt...">
+              <FontAwesomeIcon fixedWidth icon={faEdit} />
+            </Button>
+          </LinkContainer>
```

Comment:
I definitely remember the interaction here being messy when I was doing the react-router v5 upgrade, but I don't remember anything beyond that it was messy, so I guess if this works then :+1:

Reply:
Yeah it seems to work and doesn't upset the typechecker, so I figured this was good enough. :)

I do think that the types are wrong because `as={Link}` works just fine at runtime, but I also didn't want to block doing this upgrade on those types being fully correct.